File: ACE Hardware/ACE Hardware/LoginForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using MySql.Data.MySqlClient;

namespace ACE_Hardware
{
    public partial class LoginForm : Form
    {

        private MySqlConnection conn = new MySqlConnection();
        private string connstring = Program.connstring;
        public string type;
        public string username;

        public LoginForm()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {

            try
            {
                conn.ConnectionString = connstring;
                conn.Open();

                MySqlCommand comm = new MySqlCommand();
                string query = "SELECT * FROM usertbl WHERE username='" + TextBox1.Text + "' AND userpassword='" + TextBox2.Text + "'";
                MySqlDataReader reader;

                comm.Connection = conn;
                comm.CommandText = query;
                reader = comm.ExecuteReader();

                int counter = 0;

                while (reader.Read())
                    counter = counter + 1;
                if (counter > 0)
                {
                    type = reader["usertype"].ToString();
                    username = reader["username"].ToString();
                    Program.mainFormInstance.Show();
                    this.Hide();
                }
                else
                    MessageBox.Show("Please check your username and password");
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.ToString());
            }
            finally
            {
                conn.Close();
            }
            if (type == "Employee")
            {
                Program.mainFormInstance.Button1.Enabled = true;
                Program.mainFormInstance.Button2.Enabled = false;
                Program.mainFormInstance.Button3.Enabled = false;
                Program.mainFormInstance.Button4.Enabled = false;
                Program.mainFormInstance.Button5.Enabled = false;
                Program.mainFormInstance.Button6.Enabled = true;

                try
                {
                    conn.ConnectionString = connstring;
                    conn.Open();

                    MySqlCommand comm = new MySqlCommand();
                    string query = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" + username + "','Login','Logged in',NOW())";

                    comm.Connection = conn;
                    comm.CommandText = query;
                    comm.ExecuteNonQuery();
                }
                catch (Exception ex)
                {
                    MessageBox.Show(ex.ToString());
                }
                finally
                {
                    conn.Close();
                }
            }
            else if (type == "Administrator")
            {
                Program.mainFormInstance.Button1.Enabled = true;
                Program.mainFormInstance.Button2.Enabled = true;
                Program.mainFormInstance.Button3.Enabled = true;
                Program.mainFormInstance.Button4.Enabled = true;
                Program.mainFormInstance.Button5.Enabled = true;
                Program.mainFormInstance.Button6.Enabled = true;

                try
                {
                    conn.ConnectionString = connstring;
                    conn.Open();

                    MySqlCommand comm = new MySqlCommand();
                    string query = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" + username + "','Login','Logged In',NOW())";

                    comm.Connection = conn;
                    comm.CommandText = query;
                    comm.ExecuteNonQuery();
                }
                catch (Exception ex)
                {
                    MessageBox.Show(ex.ToString());
                }
                finally
                {
                    conn.Close();
                }
            }
        }
    }
}

File: POS/POS/LoginForm.vb
Imports MySql.Data.MySqlClient

Public Class LoginForm

    Dim conn As New MySqlConnection
    Dim connstring As String = "Server=localhost;User Id=root;Password=;Database=pos"
    Public type As String
    Public username As String

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Try
            conn.ConnectionString = connstring
            conn.Open()

            Dim comm As New MySqlCommand
            Dim query As String = "SELECT * FROM usertbl WHERE username='" & TextBox1.Text & "' AND userpassword='" & TextBox2.Text & "'"
            Dim reader As MySqlDataReader

            comm.Connection = conn
            comm.CommandText = query
            reader = comm.ExecuteReader

            Dim counter As Integer = 0

            While reader.Read
                counter = counter + 1
            End While
            If counter > 0 Then
                type = reader("usertype").ToString
                username = reader("username").ToString
                MainForm.Show()
                Me.Hide()
            Else
                MsgBox("Please check your username and password")
            End If
        Catch ex As Exception
            MsgBox(ex.ToString)
        Finally
            conn.Close()
        End Try
        If type = "Employee" Then
            MainForm.Button1.Enabled = True
            MainForm.Button2.Enabled = False
            MainForm.Button3.Enabled = False
            MainForm.Button4.Enabled = False
            MainForm.Button5.Enabled = False
            MainForm.Button6.Enabled = True

            Try
                conn.ConnectionString = connstring
                conn.Open()

                Dim comm As New MySqlCommand
                Dim query As String = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" & username & "','Login','Logged in',NOW())"

                comm.Connection = conn
                comm.CommandText = query
                comm.ExecuteNonQuery()
            Catch ex As Exception
                MsgBox(ex.ToString)
            Finally
                conn.Close()
            End Try

        ElseIf type = "Administrator" Then
            MainForm.Button1.Enabled = True
            MainForm.Button2.Enabled = True
            MainForm.Button3.Enabled = True
            MainForm.Button4.Enabled = True
            MainForm.Button5.Enabled = True
            MainForm.Button6.Enabled = True

            Try
                conn.ConnectionString = connstring
                conn.Open()

                Dim comm As New MySqlCommand
                Dim query As String = "INSERT INTO logstbl(username,process,description,dateandtime) VALUES('" & username & "','Login','Logged In',NOW())"

                comm.Connection = conn
                comm.CommandText = query
                comm.ExecuteNonQuery()
            Catch ex As Exception
                MsgBox(ex.ToString)
            Finally
                conn.Close()
            End Try
        End If
    End Sub

End Class